Formula/protobuf@3.6.rb

```diff
@@ -15,6 +15,8 @@ class ProtobufAT36 < Formula
 
   keg_only :versioned_formula
 
+  disable! date: "2021-02-19", because: :unmaintained
```

Comment:
Let's set this in the future (a year from now, perhaps) so that we have a deprecation period before disabling the formula.

Reply:
FYI, `protobuf@3.7` has been disable https://github.com/Homebrew/homebrew-core/blob/master/Formula/protobuf@3.7.rb

I think `protobuf@3.6` should also be disable.

Reply:
Yes, we probably should've disabled `protobuf@3.6` before `protobuf@3.7`, but we definitely should not disable formulae that still work without deprecating them first.

Reply:
Use `deprecate` instead of `disable`.